packages/Collapsible/stories/Collapsible.stories.mdx

```diff
@@ -0,0 +1,15 @@
+import { Meta, Story } from "@storybook/addon-docs/blocks";
+import { Props, Description } from "@storybook/addon-docs/blocks";
+import { Gap } from "storybook/assets/styles/common.styles";
+import Readme from "../README.md";
+import Collapsible from "../src";
+
+<Meta title=" | Collapsible/ Docs" component={Collapsible} parameters={{ options: { showPanel: false } }} />
+
+## &lt;Collapsible&gt; Props
+
+<Props of={Collapsible} />
+
+<Gap />
+
+<Description markdown={Readme} />
```

Comment:
The props in the readme are duplicating the props above, any plan to change that?

Reply:
Removed duplicates